index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="shortcut icon"
+      href="./Images/service_camp.jpeg"
+      type="image/x-icon"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Service Camp</title>
+  </head>
+  <body>
+    <main>
+      <nav class="social">
+        <ul class="social-container">
+          <li class="social-item"><a href=""></a></li>
+          <li class="social-item"
+            ><a href=""><i class="fa-brands fa-facebook-square"></i></a
+          ></li>
+          <li class="social-item"><a href="">logout</a></li>
+        </ul>
+      </nav>
+      <nav class="navbar">
+        <div class="menu">
+          <a href="./index.html"
+            ><img src="./Images/nomiddlemen.png" alt=""
```

Comment:
- [ ] Optional
 your mobile view:
![image](https://user-images.githubusercontent.com/60612329/163635750-006896d7-a981-41cf-8861-834b3b8b2d25.png)

I strongly suggest you reduce the font size of the text and reduce the characters to improve the UX in the webpage